.gitignore

```diff
@@ -4,6 +4,7 @@ irods-basicGUI/logs/*
 irods-iBridgesGui.spec
 *DS_Store
 docs/source/api/generated
+docs/outpudir
```

Comment:
Why is this there? Also, spelling error?

Reply:
Indeed ... spelling error... should be outputdir

Reply:
How does it get generated? I have never seen it come up spontaneously?